charts/prometheus-node-exporter/Chart.yaml

```
@@ -6,7 +6,7 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.33.0
+version: 4.33.1
```

A new feature should raise the minor number.

```suggestion
version: 4.34.0
```